docs/modules/ROOT/pages/guides/frameworks/nextjs.adoc

```diff
@@ -0,0 +1,54 @@
+[[usage-nextjs]]
+
+:sourcedir: example$code-partials/nextjs
+
+= Usage with Next.js
+
+This page describes the basics on how to use `@neo4j/graphql` as part of a https://nextjs.org[Next.js] fullstack application.  
```

Comment:
Maybe fullstack is a bit redundant if already talking about next.js :thinking: ?
```suggestion
This page describes the basics on how to use `@neo4j/graphql` as part of a https://nextjs.org[Next.js] application.  
```